spec/brcobranca/remessa/cnab400/itau_spec.rb

```diff
@@ -157,6 +157,9 @@
         expect(detalhe[62..69]).to eq '00000123'                       # nosso numero
         expect(detalhe[120..125]).to eq Date.today.strftime('%d%m%y')  # data de vencimento
         expect(detalhe[126..138]).to eq '0000000019990'                # valor do titulo
+        expect(detalhe[142..146]).to eq '00000'                        # agência cobradora
```

Comment:
Line is too long. [90/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.